### PR TITLE
Free/Busy

### PR DESCRIPTION
**Pull request by georgehrke**
Related to #39 

6073742C-3157-41C9-8F58-E0839A686281

### CODE REVIEW

**Comment by georgehrke** [comment]
Build of branch for testing: [calendar.tar.gz](https://github.com/nextcloud/calendar/files/3970580/calendar.tar.gz)

**Comment by georgehrke** [comment]
@tcitworld Are you fine if we just merge this into master and keep improving in smaller iterations / PRs?

File: src/components/Editor/Invitees/InviteesList.vue [hunk 1]
```diff
@@ -43,12 +43,26 @@
...
 		<OrganizerNoEmailError
 			v-if="!isReadOnly && isListEmpty && !hasUserEmailAddress" />
-		<button
-			v-if="isCreateTalkRoomButtonVisible"
-			:disabled="isCreateTalkRoomButtonDisabled"
-			@click="createTalkRoom">
-			{{ $t('calendar', 'Create Talk room for this event') }}
-		</button>
+
+		<div style="width: 100%; display: flex; justify-content: space-between; align-items: center; margin-top: 20px;">
```
Comment (georgehrke) [inline src/components/Editor/Invitees/InviteesList.vue]: Will fix this before merging

**Comment by tcitworld** [comment]
> @tcitworld Are you fine if we just merge this into master and keep improving in smaller iterations / PRs?

:+1:

**Comment by georgehrke** [comment]
Removed the inline styles, let's wait for CI and get this in.

**Comment by georgehrke** [comment]
Travis is happy, let's get this in

**Pull request merged**